code/modules/clothing/glasses/_glasses.dm

```diff
@@ -551,3 +551,15 @@
 		xray = !xray
 		var/mob/living/carbon/human/H = user
 		H.update_sight()
+
+/obj/item/clothing/glasses/sunglasses/osi
+	name = "O.S.I. Sunglasses"
+	desc = "There's no such thing as good news! Just bad news and... weird news.."
+	icon_state = "osi_glasses"
+	inhand_icon_state = "osi_glasses"
+
+/obj/item/clothing/glasses/sunglasses/phantom
+	name = "Phantom Thief Mask"
+	desc = "Lookin' cool."
+	icon_state = "phantom_glasses"
+	inhand_icon_state = "phantom_glasses"
```

Comment:
Because these are subtypes of sunglasses, they can be used in construction of new HUD sunglasses.

There's 40 of these in the dorms clothesdrobe, soo...

Reply:
I'll have to adjust them to not be sunglasses, then.

Reply:
being subtypes of sunglasses (flash proof, obviously) is wicked powerful in general yeah